app/views/tag/_subscribe_button.html.erb

```diff
@@ -1,13 +1,11 @@
 <!-- This button is used to display tags in a popover and subscribe to multiple tags using a single button -->
 <% if current_user %>
-  <a class="btn btn-default btn-xs sub" rel="popover" data-container="body" data-placement="bottom" data-template='<div class="popover" role="tooltip"><div class="arrow"></div><div class="popover-body"><%= render partial: 'tag/tags_popover', locals: {tags: tags, label_name: 'label-info'} %></div></div>' data-html="true" data-content="sample" > <i class="fa fa-plus"></i> Subscribe</a>
+  <a class="btn btn-primary btn-xs sub" rel="popover" data-container="body" data-placement="bottom" data-template='<div class="popover" role="tooltip"><div class="arrow"></div><div class="popover-body"><%= render partial: 'tag/tags_popover', locals: {tags: tags} %></div></div>' data-html="true" data-content="sample" > <i class="fa fa-plus fa-white"></i> Subscribe>> </a>
```

Comment:
Can you add a screenshot for exact same tags for 
![image](https://user-images.githubusercontent.com/20972099/51206890-48613c00-192f-11e9-90a9-25bba6e2e737.png)
?
It should have exact same colors. I want to verify it. I think your changes will not make tags blue if they are powertags. Powertags are special tags at public lab with a colon in between them.

Reply:
![image](https://user-images.githubusercontent.com/40794215/51207210-3207b000-1930-11e9-81e5-b462e4182cff.png)
 Is this okay?

Reply:
Whenever you create tags, try to think how hard it would be if you need to check the functionality yourself with random alphabets. It is far more convenient if you can always have good text in the tags. Then it will help you to check for errors.

Reply:
Yes I know that it is hard with random text, just a bit tired, but that's no excuse, should I do it again?

Reply:
Yes it is great now. Can you please commit your changes?

Reply:
This one didn't require a change, I just forgot to show you a pic of this before, probably that's why you thought this won't work, my bad.

Reply:
OK. No problem. Thanks